wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxDataSource.swift

```diff
@@ -95,7 +95,7 @@ class MessageToolboxDataSource {
      * - parameter widthConstraint: The width available to rend the toolbox contents.
      */
 
-    func updateContent(widthConstraint: CGFloat) -> SlideDirection? {
+    func updateContent(widthConstraint: CGFloat) -> Bool {
```

Comment:
What is this change?

Reply:
We already discussed that. I will update the name of the method.